nailgun/entities.py

```diff
@@ -2666,6 +2666,22 @@ def disable_rhrepo(self, base_arch,
         )
         return _handle_response(response, self._server_config, synchronous)
 
+    def list_available_rhrepos(self, reposet_id):
```

Comment:
@Ichimonji10 I've spent some time today trying to fit this name with no success. Line
```
:meth:`nailgun.entities.Product.repository_sets_id_available_repositories`
```
from doc doesn't fit in 80 chars. I tried something like this:
```
        """Call :meth:
        `nailgun.entities.Product.repository_sets_id_available_repositories`

        """
```
and this:
```
        """Call :meth:`nailgun.entities.Product.`
        `repository_sets_id_available_repositories`

        """
```
and this:
```
        """Call :meth:|method|.

        .. |method| replace::
           `nailgun.entities.Product.repository_sets_id_available_repositories`

        """
```
and few others but the result is still wrong. Couldn't find the way to break cross-references in sphinx docs either.
Can we use ```repository_sets_available_repositories``` (without ```_id_```) instead? If no, then could you help me to break that line so sphix could still understand it?